graphjet-core/src/main/java/com/twitter/graphjet/datastructures/Pair.java

```diff
@@ -1,4 +1,4 @@
-package com.twitter.graphjet.algorithms;
+package com.twitter.graphjet.datastructures;
```

Comment:
The original social proof that GraphJet returns is Map<EdgeType, LongList>, and the new social proof that GraphJet returns is Map<EdgeType, Pair<LongList, LongList>>, and here Pair usage has the lowest memory impact. Alternatively, we could create a new ConnectingUsers class which holds two LongLists, which is more clear in naming.